serverlessworkflow/modules/ROOT/pages/tooling/kn-plugin-workflow-overview.adoc

```diff
@@ -39,7 +40,12 @@ You can use the Serverless Workflow plug-in to set up your local Serverless Work
 .. Copy the `kn-workflow` binary to a directory in your `PATH`, such as `/usr/local/bin` and ensure that the file name is `kn-workflow`.
 .. On Mac, add ownership to the root user as follows:
 +
-`sudo chown root: /usr/local/bin/kn-workflow`
+`chmod +x /usr/local/bin/kn-workflow`
++
+[WARN]
+==== 
+Some systems might block the application to run due to Apple inforcing policies. To fix this problem, check the "Security & Privacy" section in the "System Preferences", "General" tab to approve the application to run. More information available at link:{apple_support_url}[this Apple support article].
```

Comment:
enforcing policies?

Reply:
Yup, you can't run any application on your mac if the dev is not certified. 😢 

Do you have a better term?

Reply:
@hmanwani-rh you might need to go over https://support.apple.com/guide/mac-help/mh40616/mac to see if the term is correct.

It's impossible here since the page opens in Portuguese, and I can't find a way to switch to EN.

Reply:
> Yup, you can't run any application on your mac if the dev is not certified. cry
> 
> Do you have a better term?

the term is fine, I was referring that I think it is "enforcing" instead "inforcing".

Reply:
@ricardozanini https://support.apple.com/guide/mac-help/mh40616/mac does not contain any word as inforce or enforce. However, _inforce_ is an obsolete spelling of _enforce_, therefore I think we should use "Apple enforcing policies" or "Apple policies".

Reply:
@hmanwani-rh I've already changed to _enforce_. Can you please do a quick review? :)